clients/python/llmengine/model.py

```diff
@@ -72,41 +73,43 @@ def create(
-                Inference framework for the LLM. Currently only DeepSpeed is supported
+                Inference framework for the LLM. Currently supported frameworks are `deepspeed`, `text_generation_inference`, `vllm`, and `lightllm`
 
             num_shards (`int`):
                 Number of shards for the LLM. When bigger than 1, LLM will be sharded
                 to multiple GPUs. Number of GPUs must be equal or larger than num_shards.
-                Only affects behavior for text-generation-inference models
 
             quantize (`Optional[Quantization]`):
-                Quantization for the LLM. Only affects behavior for text-generation-inference models
+                Quantization method for the LLM. `text_generation_inference` supports `bitsandbytes` and `vllm` supports `awq`.
 
             checkpoint_path (`Optional[str]`):
-                Path to the checkpoint for the LLM. For now we only support loading a tar file from AWS S3.
-                Safetensors are preferred but PyTorch checkpoints are also accepted (model loading will be slower).
-                Only affects behavior for text-generation-inference models
+                AWS S3 Path to the checkpoint for the LLM. LLM engine must have permission to access the given path.
+                Can be either a folder or a tar file. Folder is preferred since we don't need to untar and model loads faster.
+                For model weights, safetensors are preferred but PyTorch checkpoints are also accepted (model loading will be longer).
 
             cpus (`int`):
                 Number of cpus each worker should get, e.g. 1, 2, etc. This must be greater
-                than or equal to 1
+                than or equal to 1. Recommendation is set it to 8 * GPU count.
 
             memory (`str`):
                 Amount of memory each worker should get, e.g. "4Gi", "512Mi", etc. This must
-                be a positive amount of memory
+                be a positive amount of memory. Recommendation is set it to 24Gi * GPU count.
 
             storage (`str`):
                 Amount of local ephemeral storage each worker should get, e.g. "4Gi",
-                "512Mi", etc. This must be a positive amount of storage
+                "512Mi", etc. This must be a positive amount of storage.
+                Recommendataion is 40Gi for 7B models, 80Gi for 13B models and 200Gi for 70B models.
 
             gpus (`int`):
                 Number of gpus each worker should get, e.g. 0, 1, etc.
 
             min_workers (`int`):
                 The minimum number of workers. Must be greater than or equal to 0. This
                 should be determined by computing the minimum throughput of your workload and
-                dividing it by the throughput of a single worker. This field must be at least ``1``
-                for synchronous endpoints
+                dividing it by the throughput of a single worker. For sync or streaming endpoints,
```

Comment:
https://github.com/scaleapi/llm-engine/pull/332/files#diff-3147c6069e71e6e228995eebcaeee13595a6d2bbe8b389679c3f2905baf8090bR137

If we're saying that only `streaming` endpoints are supported, should we remove the `For sync or streaming endpoints, " phrase then?